go.mod

```diff
@@ -12,3 +12,5 @@ require (
 	golang.org/x/sync v0.0.0-20200625203802-6e8e738ad208
 	gopkg.in/yaml.v3 v3.0.0-20200615113413-eeeca48fe776 // indirect
 )
+
```

Comment:
please use go-client@f938fd9c827ecb47f38d135b7ff84ccfaecc07f3 version, we will replace that later with the release